src/main/java/com/dreammaster/gthandler/recipes/AlloySmelterRecipes.java

```diff
@@ -8,236 +11,343 @@
 import com.dreammaster.gthandler.CustomItemList;
 
 import cpw.mods.fml.common.Loader;
-import gregtech.api.enums.GT_Values;
-import gregtech.api.enums.ItemList;
-import gregtech.api.enums.Materials;
-import gregtech.api.enums.OrePrefixes;
+import gregtech.api.enums.*;
 import gregtech.api.util.GT_ModHandler;
 import gregtech.api.util.GT_OreDictUnificator;
 import gregtech.api.util.GT_Utility;
 
 public class AlloySmelterRecipes implements Runnable {
 
+    final String modIDEnderIO = "EnderIO";
```

Comment:
Mod ids should go into an enum somewhere

Reply:
I made a new enum for that now :)